src/AccessControl/tests/test_tainted.py

```diff
@@ -157,3 +180,37 @@ def testStringMethods(self):
 
     def testQuoted(self):
         self.assertEqual(self.tainted.quoted(), self.quoted)
+
+
+class TestTaintedBytes(TestTaintedString):
+
+    def setUp(self):
+        self.unquoted = b'<test attr="&">'
+        self.quoted = b'&lt;test attr=&quot;&amp;&quot;&gt;'
+        self.tainted = self._getClass()(self.unquoted)
+
+    def _getClass(self):
+        from AccessControl.tainted import TaintedBytes
+        return TaintedBytes
```

Comment:
Please add explicit tests for the constructor.

Reply:
Done